weaviate/templates/querierDeployment.yaml

```diff
@@ -0,0 +1,457 @@
+{{ if .Values.experimental.querier.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: querier
+  labels:
+    name: querier
+    app: querier
+    app.kubernetes.io/name: querier
+spec:
+  replicas: {{ .Values.replicas }}
+  updateStrategy:
+{{ toYaml .Values.updateStrategy | indent 4}}
+  serviceName: {{ .Values.experimental.querier.service.name }}-headless
+  selector:
+    matchLabels:
+      app: querier
+  template:
+    metadata:
+      labels:
+        app: querier
+        app.kubernetes.io/name: querier
+    spec:
+      securityContext:
+{{ toYaml .Values.securityContext | indent 8}}
+      {{- if (index .Values "serviceAccountName") }}
+      serviceAccountName: {{ .Values.serviceAccountName }}
+      {{- else if and (index .Values "backups" "s3" "enabled") (index .Values "backups" "s3" "serviceAccountName") }} # for backwards compatibility
+      serviceAccountName: {{ .Values.backups.s3.serviceAccountName }}
+      {{- end }}
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
+      {{- include "image.pullSecrets" (dict "pullSecrets" $.Values.image.pullSecrets) | nindent 6 }}
+      {{- include "pod.priorityClassName" ( dict "global" $.Values.globalPriorityClassName "priorityClassName" .Values.priorityClassName) | nindent 6 }}
+      containers:
+      - name: querier
+        image: '{{ .Values.experimental.querier.image}}'
+        imagePullPolicy: {{ .Values.experimental.querier.pullPolicy }}
+        {{- with .Values.experimental.querier.command }}
+        command: {{ toYaml . | nindent 10 }}
+        {{- end }}
+        {{- with .Values.experimental.querier.args }}
+        args: {{ toYaml . | nindent 10 }}
+        {{- end }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+        env:
```

Comment:
I copy pasted this from [Weaviate sts.](https://github.com/weaviate/weaviate-helm/blob/kavirajk%2Fhelm-chart-support-experimental-querier/weaviate/templates/weaviateStatefulset.yaml#L83-L85) Don't see any helpers there as well.

But this should be actually `.Values.experimental.querier.resources` instead. I will update it.

Reply:
Done.

Reply:
@jfrancoa yeah, good idea, since now we will have 2 weaviate deployments we can think of unifying those settings. I will add this task to our board @jfrancoa